_posts/2018-09-19-cockpit-178.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 178
+author: mvo
+date: 2018-09-19
+tags: cockpit linux
+slug: cockpit-178
+category: release
+summary: Fixed crashers and dropped KubeVirt support
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 178.
+
+### KubeVirt support has been removed
+
+Cockpit shipped `cockpit-kubernetes` for managing Kubernetes and OpenShift deployments in previous releases. Since its introduction, important points have come to light:
+
+1. The OpenShift Console is adding support for [KubeVirt](http://kubevirt.io/) by [merging in Tetontic and Quay](https://www.redhat.com/en/about/press-releases/red-hat-unveils-roadmap-coreos-integration-red-hat-openshift). It will become _the_ place for managing KubeVirt-based VMs that run in OpenShift.
+
+2. `cockpit-kubernetes` has been in maintenance mode for a while. It is difficult to support, as it uses a very old version of Angular.
+
+3. We haven't been able to build a recent kubevirt-enabled OpenShift image in four months. (Issues [#9479](https://github.com/cockpit-project/cockpit/issues/9479) & [#9638](https://github.com/cockpit-project/cockpit/pull/9638).) As a result, it is impossible to test `cockpit-kubernetes`.
+
+4. Community interest has waned on KubeVirt support in Cockpit.
+
+As a result, we have removed `cockpit-kubernetes` from this release.
+
+#### Migration path
+
+Fedora 29 will keep `cockpit-kubernetes`, but it will be dropped from the "rawhide" rolling release. (Rawhide is the name of the unstable version of Fedora. For the immediate future, it is currently what will become Fedora 30). Thus, `cockpit-kubernetes` will still be supported for approximately the next year-and-a-half.
+
+During this time, the Cockpit team suggests migrating to [OpenShift Console's native KubeVirt support.](https://blog.openshift.com/a-first-look-at-kubevirt/).
+
+#### Community maintenance?
+
+Recently, [`cockpit-ostree`](https://github.com/cockpit-project/cockpit-ostree) maintence and development was split out from core Cockpit. The Cockpit team would be happy to help interested developers split out `cockpit-kubernetes` into a separate upstream project too.
```

Comment:
Let's drop this paragraph from the release notes. This was a good thing to ask on the ML for discussion, but it doesn't look like this makes sense strategically.